src/core/js/views/adaptView.js

```diff
@@ -48,10 +48,18 @@ define(function(require) {
                 var model = models[i];
                 if (model.get('_isAvailable')) {
                     nthChild ++;
+
                     var ChildView = this.constructor.childView || Adapt.componentStore[model.get("_component")];
-                    var $parentContainer = this.$(this.constructor.childContainer);
-                    model.set("_nthChild", nthChild);
-                    $parentContainer.append(new ChildView({model:model}).$el);
+                    if (ChildView) {
+                        var $parentContainer = this.$(this.constructor.childContainer);
+                        model.set("_nthChild", nthChild);
+                        $parentContainer.append(new ChildView({model:model}).$el);
+                    } else {
+                        throw Error('The component \'' +
```

Comment:
Note: You don't neew the Error() syntax, can just throw a string > https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw